io.github.rahatzamancse.ProtonGameSaves.json

```diff
@@ -41,7 +41,7 @@
                 {
                     "type": "archive",
                     "path": "https://github.com/rahatzamancse/ProtonGameSaves/archive/v0.1.0.tar.gz",
-                    "sha256": "81e656723a0284be5e16ed5d734e09fb21ee33eb651d50ff2f256109472e7ced"
+                    "sha256": "c88a60da7659a681a0cbaf3f3c5abc5334ac6888489c4d2c62442cb02bb5a999"
```

Comment:
retagging is a bad habit. don't do it.

Reply:
Okay, will bump the versions from up next.